src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,210 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO `cars` (model, manufacturers_id) VALUE (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement = connection.prepareStatement(query,
+                         Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1,car.getModel());
+            preparedStatement.setLong(2,car.getManufacturer().getId());
+            preparedStatement.executeUpdate();
+            ResultSet generatedKeys = preparedStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t insert car " + car + " to DB", e);
+        }
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Car get(Long id) {
+        Car car = null;
+        String query = "SELECT c.id c_id, c.model c_model, m.id m_id, "
+                + "m.country m_country, m.name m_name FROM cars c LEFT JOIN "
+                + "manufacturers m ON m.id = c.manufacturers_id WHERE c.id = ? "
+                + "AND c.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement preparedStatement =
+                         connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCar(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get car from DB by car id " + id, e);
+        }
+        if (car != null) {
+            car.setDriverList(getDriverForCar(id));
+        }
+        return car;
+    }
+
+    @Override
+    public List<Car> getAll() {
+        List<Car> carList = new ArrayList<>();
+        String query = "SELECT c.id c_id, c.model c_model, m.id m_id,"
+                + "m.country m_country, m.name m_name FROM cars c LEFT JOIN "
+                + "manufacturers m ON m.id = c.manufacturers_id WHERE c.is_deleted = FALSE;";
```

Comment:
let's remove these prefixes `m_`, `c_`, it's redundant

Reply:
how about c_id and m_id? how to separate them